c2rust-transpile/src/cfg/mod.rs

```diff
@@ -1671,6 +1671,15 @@ impl CfgBuilder {
                     Ok(sub_stmt_next.map(|l| self.new_wip_block(l)))
                 }
 
+                CStmtKind::Attributed { substatement, ..} => {
+                    // Note: we only support the fallthrough attribute for which no action is
+                    // required.
+                    match translator.ast_context.index(substatement).kind {
+                        CStmtKind::Empty =>  Ok(Some(wip)),
+                        _ => panic!("Expected non-empty attributed statement"),
```

Comment:
This panic message is confusing--it seems like we actually expect empty attributed statements, not non-empty ones.